react/src/index.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { BrowserRouter } from 'react-router-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
```

Comment:
Style na sam dół, a `reportWebVitals` nad `App`.

Reply:
Od początku mi to nie pasowało, ale nie wiedziałem jak dokładnie to ułożyć. Dzięki!